src/main/frontend/ui.cljs

```diff
@@ -925,25 +923,19 @@
           [:div.h-2.bg-base-4.rounded.col-span-1]]
          [:div.h-2.bg-base-4.rounded]]]]])])
 
-(rum/defcs lazy-visible <
-  (rum/local false ::visible?)
-  (rum/local true ::active?)
-  [state content-fn sensor-opts {:keys [once?]}]
-  (let [*active? (::active? state)]
-    (if (or (util/mobile?) (mobile-util/native-platform?))
-      (content-fn)
-      (let [*visible? (::visible? state)]
-        (visibility-sensor
-         (merge
-          {:on-change (fn [v]
-                        (reset! *visible? v)
-                        (when (and once? v)
-                          (reset! *active? false)))
-           :partialVisibility true
-           :offset {:top -300
-                    :bottom -300}
-           :scrollCheck true
-           :scrollThrottle 500
-           :active @*active?}
-          sensor-opts)
-         (lazy-visible-inner @*visible? content-fn))))))
+(rum/defc lazy-visible
+  [content-fn]
+  (let [[hasBeenSeen setHasBeenSeen] (rum/use-state false)
+        inViewState (useInView {:rootMargin "0px 0px 0px 0px"
```

Comment:
useInView requires a js object, {:rootMargin ...}  needs a `#js` metadata, e.g. `#js {:rootMargin}`.

Reply:
Ah, ty!

BTW, I think I mean to put this back at `"300px 0px 300px 0px"` so it loads things a bit before you see them. Didn't expect you to merge this so fast!